lib/cors-anywhere.js

```diff
@@ -331,6 +332,7 @@ var getHandler = exports.getHandler = function(options, proxy) {
 // Creator still needs to call .listen()
 exports.createServer = function createServer(options) {
   if (!options) options = {};
+  help_text = null;
```

Comment:
This is not the way to fix get the second test to pass, because it prevents two proxy servers from running at the same time with different parameters.

One way to fix it is to replace the `help_text` variable with `help_texts`,

- `var help_text;` -> `var help_texts = {};`
- `help_text` -> `help_texts[help_file]`